web/html-forms.md

```diff
@@ -0,0 +1,206 @@
+# HTML Forms
+
+### Projected Time
+
+1.5 hours
+
+- Lesson: 30min
+- Guided Practice: 1hour
+- Independent Practice: 2hour
+
+### Prerequisites
+
+- [HTML lesson](./html.md)
+
+### Motivation
+
+- Forms are one of the essential topics in HTML.
+- They are widely used in almost all websites in order to gather the required information from users.
+
+### Objectives
+
+- How to create HTML forms.
+- How to add validation in the attributes.
+- How to submit the form.
+
+### Lesson
+
+HTML Form is one of the very useful aspect which has vaste usage when some information about the user has to be stored. It can be said as a document where different kind of information such as name, email, blood group, contact number and other such information can be stored.
```

Comment:
```suggestion
HTML Form is one of the most popular ways to gather user information and send it to to an application. It is also the most correct. For example, it can be used to collect things like a name and email in a sign-up form.
```